roles/openshift_metering/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+#
+# Installs OpenShift Metering
+#
+
+- name: Install OpenShift Metering block_device
+  block:
+    - name: Run OpenShift Metering install playbook
+      shell: |
+        set -o pipefail
+        . /home/cloud-user/keystonerc
```

Comment:
Please use {{ ansible_user_dir }} for all cases of `/home/cloud-user` should we ever change the user it will be more flexible.

Reply:
What is the issue with using a hard coded path before I change it?  I do not foresee the user changing, has this occurred before?

I would like to understand the pros of replacing this with a var rather than leaving it hard coded.  I see the advantage of the hard coded path revealing exactly where something comes from without the load of "unraveling" a var when doing any sort of debugging here.  In addition it can be easily cut and pasted to be run in a one off instance, where as if there is a var, you now have to determine what the value of that should be.

Reply:
I changed it for this repo, but I still don't believe the question was answered.  Has the user changed before?